os/net/netdev/netdev_ioctl.c

```diff
@@ -1154,6 +1190,40 @@ int lwip_func_ioctl(int cmd, void *arg)
 		in_arg->req_res = ret;
 		ret = 0;
 		break;
+#endif
+#if defined(CONFIG_NET_LWIP_DHCP)
+	case DHCPCSTART:
+		ret = netdev_dhcp_client_start((const char*)in_arg->host_name);
+		if (ret != 0) {
+			printf("start dhcp fail\n");
+		}
+		in_arg->req_res = ret;
+		break;
+	case DHCPCSTOP:
+		netdev_dhcp_client_stop((const char*)in_arg->host_name);
+		in_arg->req_res = 0;
+		break;
+	case DHCPDSTART:
+		ret = netdev_dhcp_server_start((char*)in_arg->host_name, _dhcpd_join);
+		if (ret != 0) {
+			printf("start dhcpd fail\n");
+		}
+		in_arg->req_res = ret;
+		break;
+	case DHCPDSTOP:
+		ret = netdev_dhcp_server_stop((char*)in_arg->host_name);
+		if (ret != 0) {
+			printf("stop dhcpd fail\n");
+		}
+		in_arg->req_res = ret;
+		break;
+	case DHCPDSTATUS:
+		ret = netdev_dhcp_server_status((char*)in_arg->host_name);
```

Comment:
Let's put a space after type.
`(char *)`

Reply:
Applied. Thanks for the comment.